### PR TITLE
[15.0] made bugs fixes in module ipushp

### PR DESCRIPTION
**Pull request by nirmal-serpentcs**
fixed not  redirecting to thank you page, 
added new thank you page , made changes for not showing existing category

### CODE REVIEW

**Comment by ketanserpentcs** [comment]
Testing has been done from Pooja Patel

**Comment by JayVora-SerpentCS** [comment]
Please make sure Chirag has reviewed.

**Comment by ChiragPatel-SerpentCS** [comment]
> Please make sure Chirag has reviewed.

Yes, I will review on this PR soon and update you.

File: ipushp/views/website_ipushp_template.xml [hunk 1]
```diff
@@ -353,47 +358,23 @@
         <t t-call="website.layout">
             <div id="wrap">
                 <div class="oe_structure" />
+                <link rel="stylesheet"
+                    href="https://2-22-4-dot-lead-pages.appspot.com/static/lp918/min/default_thank_you.css" />
```
Comment (ChiragPatel-SerpentCS) [inline ipushp/views/website_ipushp_template.xml]: @nirmal-serpentcs Instead of calling the external CSS. load those files in the module.

**Review by ChiragPatel-SerpentCS — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
we need same for v16 v17!! is that done?

**Comment by ChiragPatel-SerpentCS** [comment]
> we need same for v16 v17!! is that done?

Yes, All those changes are also done in v16 and v17.

**Pull request merged**